lambda/voting/scripts/registerAndVote.mjs

```diff
@@ -0,0 +1,125 @@
+import fetch from "node-fetch";
+import env from "dotenv";
+import { exit } from "process";
+
+import { solidityKeccak256 } from "ethers/lib/utils.js";
+import {
+  getTransactionStatus,
+  getUserWallet,
+  sleep,
+  checkAPIKey,
+} from "./utils.mjs";
+
+async function invoke(functionName, functionArgs) {
+  const url = "https://api.metakeep.xyz/v2/app/lambda/invoke";
+
+  const headers = {
+    Accept: "application/json",
+    "Content-Type": "application/json",
+    "x-api-key": process.env.API_KEY,
+    "Idempotency-Key": "Idempotency-Key" + Math.floor(Math.random() * 10000),
+  };
+  const requestBody = {
+    lambda: process.env.LAMBDA_ADDRESS,
+    function: {
+      name: functionName,
+      args: functionArgs,
+    },
+    reason: "vote",
+  };
+  const options = {
+    method: "POST",
+    headers: headers,
+    body: JSON.stringify(requestBody),
+  };
+
+  const result = await fetch(url, options);
+  const resultJson = await result.json();
+  console.log(resultJson);
+  return resultJson;
```

Comment:
exit if result not OK just like other functions

Reply:
done